src/dms-preview/azext_dms/_help.py

```diff
@@ -5,6 +5,100 @@
 
 from knack.help_files import helps
 
+helps['dms task'] = """
+    type: group
+    short-summary: Manage service-level tasks for a Database Migration Service instance.
+    long-summary: |
+        Service-level tasks are non-migration tasks that are performed at the service level. This is usually for administrative and maintenance work which applies to the service's agent and VM.
+"""
+
+helps['dms task cancel'] = """
+    type: command
+    short-summary: Cancel a service-level Task if it's currently queued or running.
+"""
+
+helps['dms task create'] = """
+    type: command
+    short-summary: Create and start a service-level task.
+    long-summary: |
+        The following tasks are currently supported:
+            1) Check OCI driver against Oracle source to check for compatibility
+            2) Upload OCI driver install package
+            3) Install an already uploaded OCI driver package
+
+    parameters:
+        - name: --task-type
+          type: string
+          short-summary: >
+            The type of task to create. The supported types are CheckOciDriver, UploadOciDriver, and InstallOciDriver.
```

Comment:
can you add parameter definition in _params.py file and use get_enum_type() for allowed values?

Reply:
The check for allowed values is handled in get_service_task_properties on line 493. It is being checked there because other things have to happen depending on task type so it made sense to validate it there as well.

Reply:
But get_enum_type() will give many convenience to users because it will make the parameters value case-insensitive.

Reply:
I'm going to push back on this. The task type is already case-insensitive. Also, as with the three state ask, we have enum-like things sprinkled elsewhere in the core and extension without using enums. So if this is important, I again would like to make the recommendation that we do any work like this in bulk rather than have the code base be inconsistent.

Reply:
I really agree that we should keep consistent. From CLI command guideline, we recommend to use get_enum_types() for enum models. For more information, you can refer to https://github.com/Azure/azure-cli/blob/dev/doc/authoring_command_modules/authoring_commands.md#registering-enums.
Besides case insensitive, get_enum_type will also help format your help message, e.g. 
 
![image](https://user-images.githubusercontent.com/49508232/75208739-8037e580-57b7-11ea-81b3-3b811dfa4002.png)

And when you have enum models, you can get enum values directly from SDK difinition.

Because your PR is in extension now, I am fine for current behavior. But when you want to make it in azure-cli repo, we hope you can consider using get_enum_type() for your enum type.